app/api/ses/webhook/route.ts

```diff
@@ -124,7 +127,8 @@ export async function POST(request: Request) {
       const s3Response = await s3Client.send(getObjectCommand);
       if (s3Response.Body) {
         const rawEmailBody = await s3Response.Body.transformToString();
-        emailBody = cleanEmailContent(rawEmailBody);
+        const mimeProcessedBody = parseMimeEmail(rawEmailBody);
+        emailBody = cleanEmailContent(mimeProcessedBody);
```

Comment:
I don't care about this being separate methods. Turn this into one called `parseEmailBody`